Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean

```diff
@@ -1,7 +1,7 @@
 /-
 Copyright (c) 2022 Chris Birkbeck. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Chris Birkbeck, David Loeffler
+Authors: Chris Birkbeck, David Loeffler, Gareth Ma
```

Comment:
By the way, I don't know if making such small changes qualify as a co-author, and I'm happy to remove this.

Reply:
I usually wouldnt for something this small (otherwise every file would have Yael's name as an author lol). But I dont know what the rules are about this, I essentially would never do it unless substantially changed/expanded the file.